recipes/geoval/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "geoval" %}
+{% set version = "0.1.5" %}
+{% set sha256 = "7da4a1879f78bb7aac2d41b5c1ee360ff74087bf857c81c532deb8cc31feb36" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py>27]
+
+requirements:
+  build:
+    - python
+    - cython
+    - numpy
```

Comment:
added x.x.   However, this still puzzles me when I am supposed to use `numpy` only and when it should be `numpy x.x`. Could not really figure out the difference from the documentation.